pkg/controllers/user/helm/common/common.go

```diff
@@ -179,10 +179,13 @@ func GenerateAnswerSetValues(app *v3.App, tempDir *HelmPath, extraArgs map[strin
 			if _, ok := extraArgs[k]; ok {
 				continue
 			}
-			values = append(values, fmt.Sprintf("%s=%s", k, v))
+			// helm will only accept escaped commas in values
+			escapedValue := escapeCommas(v)
+			values = append(values, fmt.Sprintf("%s=%s", k, escapedValue))
```

Comment:
seems pointless to set a variable to just pass it, just pass escapeCommans(v) as the 3rd param to sprintf since there is no error checking or anything. minor and not necessary for lgtm fi you want to ignore but throwing it out there.

Reply:
Had this same thought but assuming it was resolved.